src/Test/Utilities/Portable/TestResource.resx

```diff
@@ -300,6 +300,7 @@ namespace My
                 0f,
                 1.1f
             };
+            var (tuple, _) = (named: 1, 2);
```

Comment:
Yes. The test here https://github.com/dotnet/roslyn/blob/master/src/Compilers/VisualBasic/Test/Semantic/Diagnostics/DiagnosticAnalyzerTests.AllInOne.vb#L23 calls the same method as the C# one, and already has an empty HashSet. However, I just noticed this line https://github.com/dotnet/roslyn/blob/master/src/Test/Utilities/Portable/Diagnostics/TrackingDiagnosticAnalyzer.cs#L63 which apparently has `TupleType|TupleElement|TupleExpression.*` and so was ignoring the lack of tuples in the VB test, as I don't think VB has anything named like `SyntaxKind.SingleVariableDesignation` and friends, only `SyntaxKind.TupleExpression` and friends.

I'll push a new commit addressing that soon.